sdk/core/azure-core/tests/test_tracing_context.py

```diff
@@ -49,3 +49,7 @@ def work():
         span = tracing_context.current_span.get()
         assert span == current_span
         assert getattr(span, "in_worker", False)
+
+
+if __name__ == "__main__":
```

Comment:
We use pytest to drive tests.

Reply:
This is needed for the breakpoints to work in the IDEs. I can get rid of it though if you want!

Reply:
That annoying problem is caused by coverage. Running pytest with `--no-cov` solves it.

Reply:
done